gulp-tasks/test.js

```diff
@@ -242,6 +242,26 @@ function findBadMDExtensions() {
 }
 
 /**
+ * Chec to ensure there an o JavaSCript files in the content/ folder.
+ *
+ * Note: the promise will always be fulfilled, even if it fails.
+ *
+ * @return {promise} An empty promsie
+ */
+ function findJavaSCriptFiles() {
+  return new Promise(function(resolve, reject) {
+    var contentPath = GLOBAL.WF.src.content;
```

Comment:
Need to add support for `testPath`, which is used when only doing a single language.

You can probably do something like:
```
var contentPath = GLOBAL.WF.src.content;
if (GLOBAL.WF.options.testPath) {
  contentPath = GLOBAL.WF.options.testPath;
}